addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -424,7 +424,7 @@ img.o_we_custom_image.mx-auto {
 }
 
 .o_we_crop_widget {
-    background-color: #8888;
+    background-color: #888;
```

Comment:
How did that become related to current PR?

Reply:
Had a scss crash in tests without this change. I think it has to do with libsass version, @jgi-odoo also had issue locally. I think we have to make this change because I asked some months ago to upgrade libsass version (to enable more grid layout features), but it has been refused due to debian not having an up-to-date package.

Reply:
Me neither, since julien had it too, it may have appeared from last rebase master